recipes/libest/all/conanfile.py

```diff
@@ -1,80 +1,100 @@
 import os
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualRunEnv
+from conan.tools.files import apply_conandata_patches, chdir, copy, export_conandata_patches, get, replace_in_file
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.53.0"
 
 
 class LibEstConan(ConanFile):
     name = "libest"
-    license = "BSD-3-Clause"
     description = "EST is used for secure certificate enrollment"
-    topics = ("conan", "EST", "RFC 7030", "certificate enrollment")
-    homepage = "https://github.com/cisco/libest"
+    license = "BSD-3-Clause"
     url = "https://github.com/conan-io/conan-center-index"
-    settings = "os", "compiler", "build_type", "arch"
-    exports_sources = "patches/**"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
-
-    _autotools = None
+    homepage = "https://github.com/cisco/libest"
+    topics = ("EST", "RFC 7030", "certificate enrollment")
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
 
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
-        if self.settings.os in ("Windows", "Macos"):
-            raise ConanInvalidConfiguration(
-                "Platform is currently not supported by this recipe")
+        if self.settings.os == "Windows" or is_apple_os(self):
+            raise ConanInvalidConfiguration("Platform is currently not supported by this recipe")
         if self.options.shared:
-            del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("openssl/1.1.1q")
+        self.requires("openssl/1.1.1w", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
        self.requires("openssl/[>=1.1 <4]", transitive_headers=True, transitive_libs=True)
```

In case it does not support OpenSSL 3.x, you still can use range, but `[>=1.1 <3]`

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

Reply:
Thanks! Using `<2` did not work for some reason.

Reply:
@uilianries Yeah, this fails on Conan v1 for some reason. Quite annoying.

Reply:
What error?

Reply:
ERROR: Version range '>=1.1 <3' from requirement 'openssl/[>=1.1 <3]' required by 'libest/3.2.0' could not be resolved in remote 'conan-center'

Only in Conan v1.

Reply:
Thank you, I'll take a look.